### ISSUE TITLE
Qualitative Data Request for Sitewide Facilities

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-team#72798]
### Data Type Requested

User Surveys

### Data Source Type

Medallia

### Date Range Start

10-01-2023

### Date Range End

12-31-2023

### Product Name(s)

Facility Locator, VA Medical Centers, Vet Centers

### Product Description

- The Facility Locator is an unauthenticated search tool which can be used to find VA facilities and services.
- VA Medical Centers websites are comprehensive health facility pages designed to meet the needs of Veterans, their families, and caregivers. The pages provide the location details needed to prepare a Veteran to visit and easy access to top tasks and digital health tools.
- Vet Centers are community-based counseling centers that provide a wide range of counseling, outreach, and referral services to eligible Veterans, active duty service members, and their families.

### Survey URL(s), key term(s), case category or type

Facility Locator: https://www.va.gov/find-locations
VA Medical Centers: see attached file for URLs
Vet Centers: see attached file for URLs

### File Delivery Format

One file for each term/URL

### Word Cloud

No

### VA Email Address(es)

jane.xiong@va.gov, suzanne.middaugh@va.gov

### Add Assignee and Labels

If not already done so, please ensure is set as the assignee for this issue and "VSP-contact-center" and "cc-data-request" are set as labels.
Please leave this field as is when submitting.
[VetCenters.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/13813260/VetCenters.xlsx)
[VAMCs.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/13813259/VAMCs.xlsx)

**Comment by newworld2616** [comment]
Hi, I have shared the requested data, please verify once you receive it. Thank you!

**Comment by xiongjaneg** [comment]
Hi, Sarifa, we received two of the three csv's requested: Vet Centers and VA Medical Centers.

We're just waiting on the Facility Locator: https://www.va.gov/find-locations

Thanks!

**Comment by newworld2616** [comment]
There were only 2 CSV files for myhealth and location.

**Comment by xiongjaneg** [comment]
Yes, the third URL was listed directly in the request above:

Survey URL(s), key term(s), case category or type
Facility Locator: https://www.va.gov/find-locations
VA Medical Centers: see attached file for URLs
Vet Centers: see attached file for URLs

**Comment by xiongjaneg** [comment]
Received, thank you very much!

**Comment by newworld2616** [comment]
Of course!
